tuftool/README.md

```diff
@@ -1,5 +1,11 @@
 **tuftool** is a Rust command-line utility for generating and signing TUF repositories.
 
+## Dependencies
+
+Make sure you have the following dependencies installed before building `tuftool`:
```

Comment:
Nit: should be `using` since building tuftool (from the tip of develop) doesn't require openssl

Reply:
> ... dependencies present on your system before installing `tuftool`:

How's that? To avoid the double use of "installing" in one sentence.